sharkclipper/api/handlers.py

```diff
@@ -91,7 +92,10 @@ def save(handler, path, temp_dir = None, data = None, out_dir = None, **kwargs):
     # Out directory name is video name plus prefix of id.
     id_prefix = data['video']['id'].split('-')[0]
     out_dir = os.path.join(out_dir, data['video']['name'] + '-' + id_prefix)
-    os.makedirs(out_dir, exist_ok = True)
+    # If directory already exists, remake to avoid conflicts.
```

Comment:
Put a newline before this.
Take a look at how squished this code block is.

It is very common to have a comment be the start of a visual code block.
The comment will usually tell us what the code block is doing.